src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const object = {
+    result: 0,
+  };
+
+  object.add = function (number) {
+    object.result += number;
+  };
+
+  object.subtract = function (number) {
+    object.result -= number;
+  };
+
+  object.multiply = function (number) {
+    object.result *= number;
+  };
+
+  object.divide = function (number) {
+    object.result /= number;
+  };
+
+  object.reset = function () {
+    object.result = 0;
+
+    return this;
```

Comment:
Use the method shorthand for defining the 'reset' method. Also, 'this' is used in the return statement, but 'this' should refer to the 'object' itself. Make sure that 'this' is bound correctly or use 'object' to maintain clarity.